.github/workflows/android.yml

```diff
@@ -14,18 +14,46 @@ on:
       - '**.md'
 
 jobs:
+  test:
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            ui:
+              - '**/ui/**'
+
+      - name: Set up JDK 11
+        if: steps.changes.outputs.ui == 'true'
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Run Android Tests
+        if: steps.changes.outputs.ui == 'true'
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 30
+          script: ./gradlew euphony:connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.package=co.euphony.ui
+          arch: x86_64
+          target: google_apis
```

Comment:
here is important part of PR :)
Through `ui condition` check, it is better than basic usage of `reactivecircus/android-emulator-runner`!